uber/site_sections/jobs.py

```diff
@@ -117,8 +120,17 @@ def form(self, session, message='', **params):
                     defaults = cherrypy.session.get('job_defaults', defaultdict(dict))
                     defaults[params['location']] = {field: getattr(job, field) for field in c.JOB_DEFAULTS}
                     cherrypy.session['job_defaults'] = defaults
-
-                raise HTTPRedirect('index?location={}#{}', job.location, job.start_time_local)
+                tgt_start_time = str(job.start_time_local).replace(" ", "T")
+                raise HTTPRedirect('index?location=' + str(job.location) + '&time=' + tgt_start_time)
+
+        if 'start_time' in params and 'type' not in params:
+            print(params['start_time'])
```

Comment:
I expect you added these print statements for debugging -- don't forget to remove them. :)